SurveyApp.DAL/EntityModels/SurveyDataModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace SurveyApp.DAL.EntityModels
+{
+    public class SurveyDataModel : SurveyBaseDataModel
+    {
+        public override int Id { get; set; }
```

Comment:
It seems that it doesn't make sense to override "Id" property and use it from SurveyBaseDataModel